### ISSUE TITLE
0.6.74 breaks envVars['Path']

### ISSUE THREAD

**Issue by malashin** [Microsoft/vscode-go#1508]
After updating **vscode-go** to 0.6.74 I could not load **gcc** anymore, even though it was preset in the %PATH% and I was able to call it from builtin terminal.

![img](https://i.imgur.com/KWYDXv1.png)

Checking if that is the case on my other PC proved that it happens right after updating **vscode-go** to 0.6.74.

I was able to pin point the issue to this code https://github.com/Microsoft/vscode-go/commit/37570abab239ffec084cd67cdbd6ce952e8c89cb:
```go
if (goroot && (<string>envVars['PATH'] || '').split(path.delimiter).indexOf(goroot) === -1) {
	envVars['PATH'] += (envVars['PATH'] ? path.delimiter : '') + goroot;
}
```

`envVars['PATH']` does not actually exist _(at least on Windows 7)_, `envVars['Path']` does.

The code can't find the `envVars['PATH']`, but is still able to update `envVars['Path']` with 
`"undefinedC:\tools\go"`. JavaScript is awesome...

**Comment by trigun117** [comment]
![image](https://user-images.githubusercontent.com/23389076/36066155-cc1c0a8a-0ead-11e8-9171-5e73b5ea2a1c.png)
Same problem after update

**Comment by ramya-rao-a** [comment]
Released an update (0.6.75) with the fix.
Thanks for reporting!
